scripts/premoderator.sh

```diff
@@ -1,18 +1,36 @@
-#!/bin/sh -eux -o pipefail
+#!/bin/sh
 # A naive premoderation script to allow Gitlab CI pipeline on a specific PRs' comment
 # Exits with 0, if the pipeline is good to go
 
-CURL_ARGS="-fs --connect-timeout 5 --max-time 5 --retry-max-time 20 --retry 4 --retry-delay 5"
+#CURL_ARGS="-fs --connect-timeout 5 --max-time 5 --retry-max-time 20 --retry 4 --retry-delay 5"
```

Comment:
Why keep it as a comment? We can always find the old `CURL_ARGS` value  in `git history` ?